### PR TITLE
Fix deprecation warning in test suite

### PR DESCRIPTION
**Pull request by jhrcek**

### CODE REVIEW

File: test/Spec.hs [hunk 1]
```diff
@@ -25,7 +26,12 @@ main :: IO ()
 main = do
     hSetEncoding stdout utf8
     hSetEncoding stderr utf8
-    dbPool <- Pool.createPool (Sql.connectPostgreSQL connectionSettings) Sql.close 10 5 10
+    dbPool <- 
+#if MIN_VERSION_resource_pool(0,4,0)
+        Pool.newPool (Pool.defaultPoolConfig ((Sql.connectPostgreSQL connectionSettings)) Sql.close 1 10)
```
Comment (nitinprakash96) [inline test/Spec.hs:31]: Are you sure? Wasn't `newPool` added in v0.3.0.0?

Comment (jhrcek) [reply test/Spec.hs:31]: It was, but at that point there was no defaultPoolConfig, which was introduces in 0.4.0.0 :)

So I'm doing minimum amount of changes to make it work warning-free with the versions we allow in cabal file.

Comment (nitinprakash96) [reply test/Spec.hs:31]: Ah I see. 👍

**Review by nitinprakash96 — APPROVED** [review]
LGTM

**Pull request merged**